src/src/PluginStructs/P159_data_struct.cpp

```diff
@@ -0,0 +1,190 @@
+// ----------------------------------------------------------------------------
+// P159 "Dust - PM1006(K) (Vindriktning)"
+// Implementation of sensor abstraction
+// 2022 By flashmark
+// ----------------------------------------------------------------------------
+#include "../PluginStructs/P159_data_struct.h"
+
+#ifdef USES_P159
+
+const __FlashStringHelper* Plugin_159_valuename(uint8_t value_nr,
+                                                bool    displayString) {
+  const __FlashStringHelper *strings[] { /*** ATTENTION: Don't change order of values as these are stored as user-selected!!! ***/
+    F("Presence"), F("Presence"),
+    F("Stationary Presence"), F("StationaryPresence"),
```

Comment:
Quite long default taskvar names.
Can't these be shortened?

Reply:
Yeah, if only to make it more readable.